### PR TITLE
[WAJIBISHA] Merge template changes

### LINKED ISSUE #56: [WAJIBISHA] Add Contact Information to the Footer
## Description
Add contact information and links to Wajibisha/PesaCheck social media

## Tasks
- [x] Add Contact details to the footer
- [x] Add social media links to the footer
- [x] Remove placeholder text from about us page
- [x] Add the contact details to about us page
- [x] Add the team images
- [x] Remove search bar
- [x] There’s a typo in the title text --> Some of the promises *We’re* Tracking.
- [x] Also, the styling (blue and bold) is different from the one on the county page (and other pages) that reads All the promises (black and lower case) - should be the same
- [x] Remove watch and congratulate button
- [x] Style dropdowns on all promises page
- [x] Increase margin betweem home page cards and testimonial

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

Update UI issues in Wajibisha

Fixes #56 

## Screenshot:

wajibishahome

wajibisha2

wajibisha-filter

image

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+rcssmin==1.0.6
+requests==2.18.4
+rjsmin==1.0.12
+Shapely==1.6.2.post1
+six==1.11.0
+SQLAlchemy==1.1.15
+unicodecsv==0.14.1
+urllib3==1.22
+wazimap==1.1.1
+whitenoise==3.3.1
```
Comment (DavidLemayian) [inline requirements-test.txt]: @nerdlet Please delete this.

**Review by DavidLemayian — CHANGES_REQUESTED** [review]

**Comment by yazminjumaali** [comment]
Hi @nerdlet (cool name!) - team page needs names, roles and maybe link to Twitter? Also images are a distorted.

**Comment by phillipahereza** [comment]
@nerdlet please update the team page screenshot

**Comment by phillipahereza** [comment]
@nerdlet please fix, the cards aren't uniform
screen shot 2018-05-18 at 16 22 03

screen shot 2018-05-18 at 16 17 42

File: requirements.txt [hunk 2]
```diff
@@ -1,5 +1,32 @@
...
+rjsmin==1.0.12
+Shapely==1.6.2.post1
+six==1.11.0
+SQLAlchemy==1.1.15
+unicodecsv==0.14.1
+urllib3==1.22
 wazimap==1.1.1
-GDAL==1.11.0
-Shapely>=1.6
+whitenoise==3.3.1
```
Comment (DavidLemayian) [inline requirements.txt:32]: Do we need all of these?

File: wajibisha/static/css/wajibisha.scss [hunk 3]
```diff
@@ -206,10 +208,9 @@ body {
   border-color: #bfd9e8 !important;
 }
 
-.team {
-  padding: 0;
-  margin: 3rem;
-  height: 230px;
+.team.crard {
```
Comment (DavidLemayian) [inline wajibisha/static/css/wajibisha.scss]: Is this correct?

File: wajibisha/static/css/wajibisha.scss [hunk 4]
```diff
@@ -538,3 +539,46 @@ img.img-width.d-block.p-0 {
...
+  height:auto;
+  max-width:100%;
+  max-height:250px; 
+}
+
+td.table-width{
+  width:8rem!important;
+}
+
+btn-group.pl-4.pr-5.tab {
```
Comment (DavidLemayian) [inline wajibisha/static/css/wajibisha.scss]: Is this correct?

File: wajibisha/templates/_footer.html [hunk 5]
```diff
@@ -35,9 +35,19 @@
...
           </li>
         </ul>
+
+        <p class="pt-4">
+          Join Our Social Media Networks:
+        </p>
+        <ul>
+          <li class="social-icons">
+            <a href="https://twitter.com/pesacheck"><i class="fa fa-twitter fa-lg fa-fw"></i></a>
+            <a href="https://www.facebook.com/pesacheck"><i class="fa fa-facebook fa-lg fa-fw"></i></a>
```
Comment (DavidLemayian) [inline wajibisha/templates/_footer.html:45]: Just confirming these aren't set in settings.

File: wajibisha/templates/_footer.html [hunk 6]
```diff
@@ -35,9 +35,19 @@
...
+          </li>
+        </ul>
       </div>
       <div class="col-md-4 col-sm-12 mt-md-0 mt-sm-5 text-left">
-        <p class="pt-3">
+        <p class="">
```
Comment (DavidLemayian) [inline wajibisha/templates/_footer.html]: Empty class needs fix.

File: wajibisha/templates/about.html [hunk 7]
```diff
@@ -16,21 +16,42 @@ <h2>About</h2>
...
+
+          <p>uganda@pesacheck.org</p>
+        </div>
       </div>
     </div>
-    {% include 'blocks/team.html' %}
-
+  </div>
     {% endblock %}
 </div>
```
Comment (DavidLemayian) [inline wajibisha/templates/about.html]: This about page looks like it's a copy of what is on Medium. Should we not point to that instead if nothing different?

**Pull request merged**